fe/fe-core/src/main/java/org/apache/doris/nereids/stats/JoinEstimation.java

```diff
@@ -73,16 +149,15 @@ public static Statistics estimate(Statistics leftStats, Statistics rightStats, J
         Statistics innerJoinStats = null;
         if (crossJoinStats.getRowCount() != 0) {
             List<Expression> joinConditions = new ArrayList<>(join.getHashJoinConjuncts());
-            joinConditions.addAll(join.getOtherJoinConjuncts());
             innerJoinStats = estimateInnerJoin(crossJoinStats, joinConditions);
         } else {
             innerJoinStats = crossJoinStats;
         }
-        // if (!join.getOtherJoinConjuncts().isEmpty()) {
-        //     FilterEstimation filterEstimation = new FilterEstimation();
-        //     innerJoinStats = filterEstimation.estimate(
-        //             ExpressionUtils.and(join.getOtherJoinConjuncts()), innerJoinStats);
-        // }
+        if (!join.getOtherJoinConjuncts().isEmpty()) {
+            FilterEstimation filterEstimation = new FilterEstimation();
+            innerJoinStats = filterEstimation.estimate(
```

Comment:
Move this logic to `estimateInnerJoin` plz

Reply:
done